addons/account/static/src/xml/legacy_account_journal_activity.xml

```diff
@@ -3,7 +3,7 @@
 <templates>
 
      <t t-name="accountJournalDashboardActivity">
-        <t t-foreach="activities" t-as="activity">
+        <t t-foreach="activities" t-as="activity" t-key="activity_index">
```

Comment:
is t-key being required now?  (seems odd) Otherwise there is not much to say about these changes from each team I suppose.